docs/utilities/gestures.md

```diff
@@ -272,178 +272,12 @@ In this example, our app listens for gestures on the `.rectangle` element. When
-private currentColor: string = 'rgba(0, 0, 255, 0.5)';
-private lastOnStart: number = 0;
-private DOUBLE_CLICK_THRESHOLD: number = 500;
-
-ngOnInit() {
-  const gesture = this.gestureCtrl.create({
-    el: this.rectangle.nativeElement,
-    threshold: 0,
-    onStart: () => { this.onStart(); }
-  });
-
-  gesture.enable();
-}
-
-private onStart() {
-  const now = Date.now();
-
-  if (Math.abs(now - this.lastOnStart) <= this.DOUBLE_CLICK_THRESHOLD) {
-    this.rectangle.nativeElement.style.setProperty('background', this.getRandomBackground());
-    this.lastOnStart = 0;
-  } else {
-    this.lastOnStart = now;
-  }
-}
-
-private getRandomBackground() {
-  const options = this.backgrounds.filter(bg => bg !== this.currentColor);
-  this.currentColor = options[Math.floor(Math.random() * options.length)];
-
-  return this.currentColor;
-}
-```
-</TabItem>
-<TabItem value="react">
-
-```javascript
-const backgrounds = ['rgba(0, 0, 255, 0.5)', 'rgba(0, 255, 0.5)', 'rgba(255, 0, 0, 0.5)', 'rgba(255, 255, 0, 0.5)', 'rgba(255, 0, 255, 0.5)', 'rgba(0, 255, 255, 0.5)'];
-const DOUBLE_CLICK_THRESHOLD = 500;
-const rectangle = document.querySelector('.rectangle');
-const gesture = createGesture({
-  el: rectangle,
-  threshold: 0,
-  onStart: () => { onStart(); }
-});
-
-gesture.enable();
-
-let lastOnStart = 0;
-let currentColor = 'rgba(0, 0, 255, 0.5)';
-
-const onStart = () => {
-  const now = Date.now();
-
-  if (Math.abs(now - lastOnStart) <= DOUBLE_CLICK_THRESHOLD) {
-    rectangle.style.setProperty('background', getRandomBackground());
-    lastOnStart = 0;
-  } else {
-    lastOnStart = now;
-  }
-}
-
-const getRandomBackground = () => {
-  const options = backgrounds.filter(bg => bg !== currentColor);
-  currentColor = options[Math.floor(Math.random() * options.length)];
-
-  return currentColor;
-}
-```
-</TabItem>
-<TabItem value="vue">
-
-```javascript
-import { createGesture } from '@ionic/vue';
-import { ref } from 'vue';
-
-...
-
-const backgrounds = ['rgba(0, 0, 255, 0.5)', 'rgba(0, 255, 0.5)', 'rgba(255, 0, 0, 0.5)', 'rgba(255, 255, 0, 0.5)', 'rgba(255, 0, 255, 0.5)', 'rgba(0, 255, 255, 0.5)'];
-const DOUBLE_CLICK_THRESHOLD = 500;
-const rectangleRef = ref();
-const gesture = createGesture({
-  el: rectangleRef.value,
-  threshold: 0,
-  onStart: () => { onStart(); }
-});
-
-gesture.enable();
-
-let lastOnStart = 0;
-let currentColor = 'rgba(0, 0, 255, 0.5)';
-
-const onStart = () => {
-  const now = Date.now();
-
-  if (Math.abs(now - lastOnStart) <= DOUBLE_CLICK_THRESHOLD) {
-    rectangleRef.value.style.setProperty('background', getRandomBackground());
-    lastOnStart = 0;
-  } else {
-    lastOnStart = now;
-  }
-}
-
-const getRandomBackground = () => {
-  const options = backgrounds.filter(bg => bg !== currentColor);
-  currentColor = options[Math.floor(Math.random() * options.length)];
-
-  return currentColor;
-}
-```
-</TabItem>
-</Tabs>
-````
+<DoubleClick />
```

Comment:
Comparing it to the animations page, let's move this after the description of the playground for consistency.